rust/gitxet/tests/integration_tests/test_git_xet_init.sh

```diff
@@ -108,10 +108,74 @@ mkdir repo_5
 pushd repo_5
 
 git init --bare
-git xet init -m 2 --force --bare
+git xet init -m 2 --force 
```

Comment:
Actually xetea runs `git xet init -m 2 --bare`, `--force` shouldn't be needed because it serves as a remote. Let's update this to reflect the same logic there.

Reply:
Ah, good catch.  I'll update force so that if it runs on a bare repo, it doesn't do the check for remotes.